money/currency.py

```diff
@@ -1,1308 +1,1136 @@
 """Module defining currencies and currency utilities"""
 
 from enum import Enum
+from typing import Any, Dict
 
-#pylint: disable=too-many-lines
+from babel.numbers import get_currency_symbol
 
-class Currency(Enum):
```

Comment:
Consolidated currency details inside Enum. Previously, the behavior was split between `Currency` and `CurrencyHelper` methods